src/info.plist

```diff
@@ -81,39 +60,77 @@
 			<key>version</key>
 			<integer>2</integer>
 		</dict>
+		<dict>
+			<key>config</key>
+			<dict>
+				<key>argumenttype</key>
+				<integer>0</integer>
+				<key>keyword</key>
+				<string>todo</string>
+				<key>subtext</key>
+				<string>todo: {query}</string>
+				<key>text</key>
+				<string>Add to Todoist Inbox</string>
+				<key>withspace</key>
+				<true/>
+			</dict>
+			<key>type</key>
+			<string>alfred.workflow.input.keyword</string>
+			<key>uid</key>
+			<string>E053B624-0E5A-40D6-BC12-28DBDBD3CE8B</string>
+			<key>version</key>
+			<integer>1</integer>
+		</dict>
 	</array>
 	<key>readme</key>
-	<string>For the workflow to work correctly, set the "token" variable.
+	<string>To use the workflow, first get a token from the [integration settings section](https://todoist.com/app/settings/integrations).
+
+To add a task to Todoist Inbox, just type `todo buy milk` and press Enter.
 
-To get a token, go to the integration settings section https://todoist.com/app/settings/integrations</string>
+![](preview.png)</string>
 	<key>uidata</key>
 	<dict>
 		<key>64240F2E-6829-4B1C-A674-9927BB1182FB</key>
 		<dict>
 			<key>xpos</key>
-			<integer>455</integer>
+			<real>455</real>
 			<key>ypos</key>
-			<integer>275</integer>
+			<real>265</real>
 		</dict>
 		<key>E053B624-0E5A-40D6-BC12-28DBDBD3CE8B</key>
 		<dict>
 			<key>xpos</key>
-			<integer>215</integer>
+			<real>215</real>
 			<key>ypos</key>
-			<integer>265</integer>
+			<real>265</real>
 		</dict>
 	</dict>
-	<key>variables</key>
```

Comment:
How do you think there is any chance to keep one workflow for both versions of Alfred (4 and 5)?

Reply:
Can’t have the same Workflow file working in both versions and keep the User Configuration, as that’s a new feature. But Alfred 5 recognises the `.alfred5workflow` extension, while Alfred 4 does not. That’s a simple way to separate Workflows if you which to support multiple versions at once.